printer_mods/arhodiewithsoul/Gantry/X_Axis/X_Carriage/readme.md

```diff
@@ -0,0 +1,8 @@
+Contains updated X Carriage mounts (and connector cover) to try assist in packaging connectors under the connector cover.
+
+![x_carriage_frame_right_V1 8_001 (Large)](https://user-images.githubusercontent.com/80538348/116250629-ca26ad80-a765-11eb-9a9f-608dbce2e904.jpg)
+![x_carriage_frame_right_V1 8_002 (Large)](https://user-images.githubusercontent.com/80538348/116250636-cb57da80-a765-11eb-81fd-3641de3dad2c.jpg)
```

Comment:
X carriage frame STLs are not in a printable orientation.  Also, why does one of them have `_V1.8` in the name and the other one doesn't?

Reply:
Carriage frames were named as per the parts here: https://github.com/VoronDesign/Voron-1/tree/Voron1.8/STLs/Gantry/X_Axis/X_Carriage

I have renamed my parts with the v1.8 suffix to both, as per your query.

Reply:
Re-oriented Carriage frames